mudlet-mapper.xml

```diff
@@ -5834,7 +5891,17 @@ function mmp.addWingsAchaea()
         word
       )
   end
-
+	
+	local function orbed()-- orb of confinement - PapaGuacamole
+		return
+				(getRoomArea(mmp.currentroom) == 49 and mmp.settings.orbashtan) or
+    		(getRoomArea(mmp.currentroom) == 57 and mmp.settings.orbcyrene) or
+    		(getRoomArea(mmp.currentroom) == 51 and mmp.settings.orbeleusis) or
+    		(getRoomArea(mmp.currentroom) == 55 and mmp.settings.orbhashan) or
+    		(getRoomArea(mmp.currentroom) == 44 and mmp.settings.orbmhaldor) or
+    		(getRoomArea(mmp.currentroom) ==368 and mmp.settings.orbtargossas)
```

Comment:
Maybe call `getRoomArea(mmp.currentroom))` only once? It shouldn't change between those calls :stuck_out_tongue:

Reply:
I have updated that in pull request 81, this one's kind of been superseded by that one (because I forked at the wrong point)

Reply:
I can't see the change there either. Did you push your branch?

Reply:
I ended up with two similar branches, with the changes not always going to the right one! https://github.com/IRE-Mudlet-Mapping/ire-mapping-script/pull/82 is the right one